frontend/src/app/modules/queue-dashboard/services/queue-dashboard.service.ts

```diff
@@ -0,0 +1,45 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpParams} from "@angular/common/http";
+import {BehaviorSubject, ReplaySubject} from "rxjs/index";
+import {WptServerDTO} from "../models/WptServerDTO";
+import {LocationInfoDTO} from "../models/LocationInfoDTO";
+
+export interface ServerInfo {
+  [serverId: number]: LocationInfoDTO[],
+}
+
+@Injectable()
+export class QueueDashboardService {
+
+  private http: HttpClient;
+
+  activeServers$ = new ReplaySubject<WptServerDTO[]>(1);
+  serverInfo$ = new BehaviorSubject<ServerInfo>({});
+
+  constructor (httpClient: HttpClient){
```

Comment:
You can use `private http: HttpClient` instead of this, the declaration above and the assignment below.